gulpfile.js

```diff
@@ -52,19 +52,23 @@ gulp.task('apidoc', (done) => {
   }, done);
 });
 
-gulp.task('test', () => {
-  return gulp.src('./dist-server/test/node/*.js')
+gulp.task('test:index', () => {
+  return gulp.src('./dist-server/test/index.js')
     .pipe(tape({
       reporter: faucet(),
     }));
 });
 
+gulp.task('test', () => {
+  runSequence('babel','test:index');
```

Comment:
So as we're adding more unit tests like test:index above, we'll need to link them all on this line? Could you add comment to do that if that's the case?

Also, weren't there more than index.js for tests? (user.js?) Before *.js was applied so all the tests were set up to run, but we now need to individually add them here.